portfolio/src/main/webapp/script.js

```diff
@@ -35,17 +35,17 @@ async function getGreeting(){
 /**
  * Fetches messages from the servers and adds them to the DOM.
  */
-async function getGreetings() {
-  fetch('/data').then(response => response.json()).then((msg) => {
+async function getComments() {
+    fetch('/data').then(response => response.json()).then((msg) => {
```

Comment:
Indentation should also be 2 spaces for Java, unless you are breaking apart a complete statement. IE the createListElement lines are OK with 4 spaces below because the statement ends with the ;

Your editor might be automatically adding 4 spaces, you should make sure it's tab is set to 2 spaces.